mkdocs.yml

```diff
@@ -20,6 +20,7 @@ nav:
       - models.md
       - dependencies.md
       - tools.md
+      - toolsets.md
```

Comment:
I feel like `common_tools` might make more sense than `toolsets` today given it's just .. one tool. But 🤷‍♂️ up to you